### DISCUSSION TITLE
Is it intentional to ship `/tests` via composer?

### DISCUSSION THREAD

**Discussion by gsteel** [PHPCSStandards/PHP_CodeSniffer#561] [Q&A] [answered]
I noticed that the tests for phpcs are present in vendor when installed as a dependency and they're not listed for `export-ignore` in `.gitattributes`. Is this intentional?

**Comment by jrfnl** [comment, marked as answer]
Yes, but they no longer will be as of version 4.0.

For context, see the previous discussions about this:
* https://github.com/squizlabs/PHP_CodeSniffer/issues/548
* https://github.com/squizlabs/PHP_CodeSniffer/issues/1908

**Reply by gsteel** [reply]
Thanks for the detailed answer 🙂
